src/main/java/com/adobe/aem/compgenerator/models/ProjectSettings.java

```diff
@@ -93,6 +104,6 @@ public void setAppsPath(String appsPath) {
     @Override
     public boolean isValid() {
         return StringUtils.isNotBlank(modelInterfacePackage) && StringUtils.isNotBlank(modelImplPackage)
-                && StringUtils.isNotBlank(componentPath) && StringUtils.isNotBlank(bundlePath) && StringUtils.isNotBlank(appsPath);
+                && StringUtils.isNotBlank(componentPath) && StringUtils.isNotBlank(bundlePath) && StringUtils.isNotBlank(testPath) && StringUtils.isNotBlank(appsPath);
```

Comment:
Hrmm...  we really shouldn't require `testPath` unless `hasTestClass` is true.  We probably have a similar issue for other features.  However, I'd like to update this at least for this new feature, so that people running with old config files can still successfully run the generator w/o adding in the new test path property.